README.md

```diff
@@ -32,6 +32,7 @@ gulp.task('test', function (cb) {
       gulp.src(['test/*.js'])
         .pipe(mocha())
         .pipe(istanbul.writeReports()) // Creating the reports after tests runned
+        .pipe(istanbul.enforceThresholds()) // Check coverage against thresholds
```

Comment:
Please add a basic configuration object. This doesn't make sense without it.

Also the comment is pretty unclear. Maybe more like:

> Enforce a coverage of at least 90%